NTupleMaker/bin/SynchNTupleProducer_2017.cpp

```diff
@@ -1971,11 +2044,20 @@ void FillGenTree(const AC1B * analysisTree, Synch17GenTree *gentree, TString ch)
     }
   }
 
-  
+  //Merijn: this is all done also in gen_acott, so don't do here..
+  /*
   gentree->acotautau_00 = -9999;
   gentree->acotautau_01 = -9999;
   gentree->acotautau_10 = -9999;
   gentree->acotautau_11 = -9999;
+
+  //Meirjn not sure why init it appears again here..
```

Comment:
Some functions are called only under certain circumstances, the computation of the acoplanarity angle was initially done only in the tau tau channel for instance, so to ensure that the variable is correctly initialized for all the events the initialization was first written here.
However for tests and debugging purposes it was useful to see the functions operate directly on a single root file in interactive mode, thus needing to repeat the initialization there.

As of now, the acoplanarity angle is computed for all events, so for all purposes the initialization can be moved to the function.